pnpm-lock.yaml

```diff
@@ -1,231 +1,415 @@
-lockfileVersion: '9.0'
```

Comment:
Hmm I think this is wrong. You have something wrong with your pnpm version

Reply:
Good catch, thanks.